Selenium_cource/Homework1_2/Change_password.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'Verification'
+
+module Change_password
+
+  #Change password
+  def change_password(password, new_password)
+    include Verification
```

Comment:
Wrong place for include
It should be placed just after opening class or module like

class SomeClass
  include SomeModule
end